### DISCUSSION TITLE
Issues with Custom Snap Areas and Long Throws in Rectangle Pro on Samsung Odyssey Display

### DISCUSSION THREAD

**Discussion by jab3z** [rxhanson/RectanglePro-Community#518] [Q&A] [answered]
I've been using Rectangle Pro and created some custom areas, which I enabled to be used as snap targets. Initially, everything worked fine, but now these areas are no longer visible on the display, nor do they function as snap targets. Additionally, I configured them as long throws, but that feature isn't working either.

Could you provide guidance on how to fix these issues? Also, if you have any resources or material on creating custom zones more in-depth, that would be greatly appreciated. This problem is occurring on an external Samsung Odyssey display.

Attached is a screenshot for reference.

Thank you!

![image](https://github.com/rxhanson/RectanglePro-Community/assets/4821049/d062ca95-8e28-4408-9bd2-7e76f2aeb4c2)

**Comment by rxhanson** [comment, marked as answer]
Thanks for reporting. The usual cause of this is if you have moved your display in System Settings, and the custom sizes/positions are no longer valid. At this time, the Rectangle Pro configuration won't auto-adjust if this happens, although this is on my backlog to fix.

You could re-record your custom size/positions or change them to be relative rather than absolute. For example, on bottom right 25%, you could set docking position: bottom right, W: 1/2, H: 1/2, and Display: 1. 

Let me know if that's not the issue you ran into or if this isn't clear.

**Comment by jab3z** [comment]
thanks @rxhanson . It works now. I also just purchased the pro.
